Makefile

```diff
@@ -36,7 +36,7 @@ MANPAGES_MD = $(wildcard docs/*.md)
 #     Note: Uses the -N -l go compiler options to disable compiler optimizations
 #           and inlining. Using these build options allows you to subsequently
 #           use source debugging tools like delve.
-all: binary docs
+all: binary binary-static docs
```

Comment:
With this, `make all` _both_ runs `make binary`→`make binary-local`, and runs `make binary-static`→`make binary-static-local`, i.e. a `skopeo` binary is created twice, with the static one overwriting the dynamic one. That doesn’t really make sense.

@hustcat What was this _intended_ to do?